src/main/scala/org/zalando/spearheads/innkeeper/services/team/TeamService.scala

```diff
@@ -81,10 +81,12 @@ class ZalandoTeamService @Inject() (
   }
 
   private def callApplicationService(applicationName: String, token: String): Future[Result[Team]] = {
-    val url = applicationServiceUrl + applicationName
+    val normalizedApplicationName =
+      if (applicationName.startsWith("stups_")) applicationName.substring("stups_".length) else applicationName
```

Comment:
can be replaced with `val normalizedApplicationName = applicationName.replaceFirst("^stups_", "")`